app/assets/javascripts/osem.js

```diff
@@ -180,6 +189,18 @@ $( document ).ready(function() {
         var max = $selected.data("max-words");
         word_count(this, 'submission-count', max);
     });
+
+    /* Listen for reset template button, wait for confirm, and reset. */
```

Comment:
The amount of comments you've in the changes you made are good but be sure to describe why you're doing each action instead of simply stating what you're doing.